apps/kyb-app/src/components/organisms/DynamicUI/StateManager/components/ActionsHandler/action-handlers/value-apply/json-logic.selector.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  JsonLogicSelector,
+  ValueApplyValue,
+} from '@/components/organisms/DynamicUI/StateManager/components/ActionsHandler/action-handlers/value-apply/types';
+import { ValueApplySelector } from '@/components/organisms/DynamicUI/StateManager/components/ActionsHandler/action-handlers/value-apply/value-apply.selector.abstract';
+import jsonLogic from 'json-logic-js';
+
+export class ValueApplyJsonLogicSelector implements ValueApplySelector {
+  select<TResult>(value: ValueApplyValue, context: any): TResult {
```

Comment:
Context can only be some variation of an object, right? If `unknown` instead of `any` in the record works even better. 🙂 
```suggestion
  select<TResult>(value: ValueApplyValue, context: Record<PropertyKey, any>): TResult {
```

Reply:
`AnyObject` fits better